torch/csrc/api/include/torch/nn/module.h

```diff
@@ -230,7 +230,7 @@ class TORCH_API Module : public std::enable_shared_from_this<Module> {
   /// \endrst
   std::vector<std::shared_ptr<Module>> modules(bool include_self = true) const;
 
-  /// Returns an `OrderedDict` of he submodules of this `Module` (the entire
+  /// Returns an `OrderedDict` of the submodules of this `Module` (the entire
   /// submodule hierarchy) and thei keys, and if `include_self` is true, also
```

Comment:
Thanks a lot for catching the typo! Would you like to also change `thei keys` to `their keys` in this line as well?

Reply:
OK. Thanks for the reply.